maine-thesis.cls

```diff
@@ -79,6 +79,18 @@
 \RequirePackage[none]{hyphenat}[2009/09/02 v2.3c hyphenation utilities]
 \AfterPackage{hyperref}{% Graduate School requirements mandate a slight modification to hyperref's redefinition of the caption command.  This is implemented here.  The command is only executed after hyperref is loaded in the document (if it is loaded).
 \long\def\@caption#1[#2]#3{%
+\hypersetup{ pdfdisplaydoctitle=true}
```

Comment:
This was actually a typo, 
 in my own edited copy of the class file these calls were in the correct place originally. Glad you spotted that.